examples/custom-resource/libraries/gordon.rb

```diff
@@ -0,0 +1,15 @@
+class Gordon < Inspec.resource(1)
+  name 'gordon'
+
+  example "
+    describe gordon do
+      its('example') { should eq 'foobar' }
+    end
+  "
+
+  def example
```

Comment:
Also: defining a method named `example` is a bit high risk - `example` is part of the resource DSL!  No collision in this case, but the unfamiliar user (for whom this code is intended) would be easily confused.